powerdns_sync/debian/atomiadns-powerdnssync.atomiadns-powerdnssync.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
```

Comment:
However weird it might seem it is the correct name, the reason being that the -name parameter to dh_installinit looks for atomiadns-powerdnssync.(name).service and uses (name) as the service name. I did'nt manage to find a smarter way to do this ie keep upstart on <16.04. Without name it selects .upstart before .service